modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
I noticed an `nl_abs_tol = 1e-4` in the test below. These loosely converged tests often fail on different platforms or different compilers. Is there a reason you can't tighten up the original test and lose the loose tolerances?